javascript/javascript2/week3/variableFunction.js

```diff
@@ -0,0 +1,29 @@
+//1-
+const array = [
+  (firstFunction = function () {
+    console.log("first function");
+  }),
+  (secondFunction = function () {
+    console.log("second function");
+  }),
+  (thirdFunction = function () {
+    console.log("third function");
+  }),
```

Comment:
The variables are never used so you can remove them.
```suggestion
  (function () {
    console.log("first function");
  }),
  (function () {
    console.log("second function");
  }),
  (function () {
    console.log("third function");
  }),
```

Reply:
thanks, I will change it